app/page-partials/edit-request/CancelConfirmModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Button from '@button-inc/bcgov-theme/Button';
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+import CenteredModal from 'components/CenteredModal';
+import CancelButton from 'components/CancelButton';
+
+interface Props {
+  onConfirm?: Function;
+}
+
+function CancelConfirmModal({ onConfirm }: Props) {
+  const modalId = 'edit-cancel-confirmation';
+
+  const handleConfirm = async () => {
+    if (onConfirm) await onConfirm();
+    window.location.hash = '#';
+  };
+
+  const openModal = () => (window.location.hash = modalId);
+
+  const modalContents = (
+    <>
+      <p>
+        Are you sure you want to press cancel?
+        <br />
+        If you continue, you will lose any data that you have provided.
+      </p>
+    </>
+  );
+
+  return (
+    <>
+      <CancelButton variant="secondary" size="medium" type="button" onClick={openModal}>
+        Cancels
```

Comment:
Should this be 'Cancel' instead of 'Cancels' ?

Reply:
good catch! I fixed the typo.